source/payment_links/index.html.md.erb

```diff
@@ -5,8 +5,9 @@ weight: 45
 
 # Payment links
 
-You may want to use payment links rather than integrate with the GOV.UK Pay
-API if your service:
+Payment links allow you to receive online payments from your users without integrating with the GOV.UK Pay API. When a user accesses your payment link, they’ll be able to make an online card payment.
```

Comment:
@katiebates Could you review lines 8-10? Following 2i we've added a short description of Payment links (lifted directly from the product guidance) to reduce the risk Alex flagged of confusing payment links with payment pages.